scripts/admin/tripGlobal.ts

```diff
@@ -0,0 +1,156 @@
+ */
+
+const addresses: DeploymentAddresses = getAllAddresses(mode);
+const testnets = process.env.npm_config_testnets == "true";
+let activeChainSlugs: string[];
+if (testnets)
+  activeChainSlugs = Object.keys(addresses).filter((c) =>
+    isTestnet(parseInt(c))
+  );
+else
+  activeChainSlugs = Object.keys(addresses).filter((c) =>
+    isMainnet(parseInt(c))
+  );
+const sendTx = process.env.npm_config_sendtx == "true";
+const integrationType = process.env.npm_config_integration
+  ? process.env.npm_config_integration.toUpperCase()
+  : IntegrationTypes.fast;
+let filterChains = process.env.npm_config_chains
+  ? process.env.npm_config_chains.split(",").map((c) => Number(c))
+  : activeChainSlugs;
+
+const main = async () => {
+  if (
+    integrationType &&
+    !Object.values(IntegrationTypes).includes(
+      integrationType as IntegrationTypes
+    )
+  ) {
+    throw new Error(
+      "Invalid integration type. Can be FAST, NATIVE_BRIDGE or OPTIMISTIC"
+    );
+  }
+  filterChains = filterChains.map((c) => Number(c));
+  console.log({ filterChains });
+  for (const chain of filterChains) {
+    let siblingChains = getSiblings(deploymentMode, Number(chain) as ChainSlug);
+
+    if (siblingChains.length)
+      console.log("======= Checking ", { chain }, "==============");
+    for (const siblingChain of siblingChains) {
+      const switchboard = getSwitchboardInstance(
+        chain,
+        siblingChain,
+        integrationType as IntegrationTypes,
+        mode
+      );
+      if (switchboard === undefined) {
+        console.log(
+          "Switchboard address not found for ",
+          chain,
+          "continuing..."
+        );
+        continue;
+      }
+
+      let tripStatus: boolean;
+      try {
+        tripStatus = await switchboard.isGlobalTipped();
+        console.log({ type: integrationType, tripStatus });
+      } catch (error) {
+        console.log("RPC Error while fetching trip status: ", error);
+        break;
+      }
+
+      if (tripStatus) break; // as global trip, check for a single siblingChain is enough
+
+      if (sendTx) {
+        let hasRole = await checkRole(ROLES.TRIP_ROLE, switchboard);
+        if (!hasRole) break;
+        console.log("tripping", { chain });
+
+        const nonce = await switchboard.nextNonce(
+          switchboard.signer.getAddress()
+        );
+
+        let sigIdentifier =
+          integrationType == IntegrationTypes.native
+            ? TRIP_NATIVE_SIG_IDENTIFIER
+            : TRIP_GLOBAL_SIG_IDENTIFIER;
+        const digest = keccak256(
+          defaultAbiCoder.encode(
+            ["bytes32", "address", "uint32", "uint256", "bool"],
+            [sigIdentifier, switchboard.address, chain, nonce, true]
+          )
+        );
+
+        const signature = await switchboard.signer.signMessage(
+          arrayify(digest)
+        );
+
+        const tx = await switchboard.tripGlobal(nonce, signature, {
+          ...overrides(chain),
+        });
```

Comment:
we should log overrides also, very easy to run this script with wrong overrides when doing in hurry

Reply:
and try to make the logs pretty instead of json